src/Elastic.Apm/Helpers/PlatformDetection.cs

```diff
@@ -33,6 +33,9 @@ internal static class PlatformDetection
 			RuntimeInformation.FrameworkDescription.Length >= 6
 			&& char.IsDigit(RuntimeInformation.FrameworkDescription[5]);
 
+		/// <summary> Checks if runtime is .NET Core or .NET 6 and up </summary>
```

Comment:
nit: `IsDotNetCore` is true for .NET 5 as well.

```suggestion
		/// <summary> Checks if runtime is .NET Core or .NET 5 and up </summary>
```